### ISSUE TITLE
Line Shuffling

### ISSUE THREAD

**Issue by Ethosa** [cherriesandmochi/gdmaim#7]
This feature can be helpful to obfuscate source code I think:

Before:
```gdscript
var some: int = 0
var other: String = "222"

func test() -> void:
    pass

class TestClass:
    func hello():
        pass
```

After, seed `NNNN`
```gdscript
class D_FsXk:
    func _xgSx_k():
        pass
var _sXsfS: int = 0
func _JSfXf8Sso() -> void:
    pass
var _xsFs: String = "222"
```

After, seed `MMMM`
```gdscript
var _sXsfS: int = 0
class D_FsXk:
    func _xgSx_k():
        pass
func _JSfXf8Sso() -> void:
    pass
var _xsFs: String = "222"
```

**Comment by Ethosa** [comment]
This can works because GDScript top-level declaration is not importantly

**Comment by Ethosa** [comment]
Some lines, that can seems like a problem:
```gdscript
var x: int = 100
var y: int = x + 100
```

In GDScript at top-level statement sequence is not important 👀
So this wotks:
```gdscript
var y: int = x + 100
var x: int = 100
```

**Comment by cherriesandmochi** [comment]
Implemented in 7faaac7.
